modules/store/spec/selector.spec.ts

```diff
@@ -562,13 +559,13 @@ describe('Selectors', () => {
 
       describe('warning will ', () => {
         it('be logged when not in mock environment', () => {
-          const ngSpy = jest.spyOn(ngCore, 'isDevMode').mockReturnValue(true);
+          const ngSpy = vi.mocked(ngCore.isDevMode).mockReturnValue(true);
           const selector = createFeatureSelector('featureB');
 
           selector({ featureA: {} });
 
           expect(warnSpy).toHaveBeenCalled();
-          expect(warnSpy.calls.mostRecent().args[0]).toMatch(
+          expect(warnSpy.mock.calls[warnSpy.mock.calls.length - 1][0]).toMatch(
```

Comment:
I think `lastCall` will be much cleaner https://vitest.dev/api/mock.html#mock-lastcall